### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+```
+    <dependency>
+    <groupId>io.github.awsjavakit</groupId>
+    <artifactId>misc</artifactId>
+    <version>0.13.2</version>
+    </dependency>
+```   
+**Important Note:**   
+We are currently supporting only the specified versions of all the libraries that 
+aws-java-kit depends on. Different versions can produce a build error on the client's end.
```
Comment (brinxmat) [inline README.md:23]: `Different versions can produce a build error on the client's end.` -> `Different versions in client code may produce build errors`

File: README.md [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+
+
+## Modules:
+All functionality is described in the test classes. Below you can find a short summary 
+of the implemented features.
+
+### Testing Utils (testingutils):
+**Description:**
+The `testingutils` module contains: 
+1. A set of simple fake clients for using during testing.
```
Comment (brinxmat) [inline README.md:33]: `for using during`-> `for use in`

File: README.md [hunk 3]
```diff
@@ -1,8 +1,46 @@
...
+## Modules:
+All functionality is described in the test classes. Below you can find a short summary 
+of the implemented features.
+
+### Testing Utils (testingutils):
+**Description:**
+The `testingutils` module contains: 
+1. A set of simple fake clients for using during testing. 
+Most clients implement only a small subset of the actual actions.
+However, it is easy to add any missing functionality.
```
Comment (brinxmat) [inline README.md:35]:
```
Most clients implement only a small subset of the actual actions.
However, it is easy to add any missing functionality.
```
-> 
```
Most clients implement only a limited subset of API methods, however, it is simple to extend the clients to add missing functionality.
```
The question is "how?"

File: README.md [hunk 4]
```diff
@@ -1,8 +1,46 @@
...
+
+### Testing Utils (testingutils):
+**Description:**
+The `testingutils` module contains: 
+1. A set of simple fake clients for using during testing. 
+Most clients implement only a small subset of the actual actions.
+However, it is easy to add any missing functionality. 
+All implemented functionality is described in the associated test class.
+2. An HttpClient that works with Wiremock and the Https protocol.
+3. A Hamcrest matcher that checks recursively whether a class has some empty field.
```
Comment (brinxmat) [inline README.md:38]: remove "some"

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (axthosarouris) [reply README.md:35]: Good point about the "how".  I will add it later.

**Review by brinxmat — APPROVED** [review]

**Pull request merged**